templates/page/team.html

```diff
@@ -20,7 +20,7 @@
 <script type="text/javascript">
   $(document).ready(function()
   {
-    $(".hyper").attr("href", "http://cerrocastillo-esp.lpe.cl/equipo");
+    $(".hyper").attr("href", "https://www.cerrocastillocapital.cl/equipo");
```

Comment:
ojo con los espacios

Reply:
si hice un find and replace, lo hago igual en todos los sitios?

Reply:
por lo menos en esa linea,  porque si modificamos, para que no pase cagando